lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -458,13 +475,7 @@ def message_from(succeeded, response)
       end
 
       def authorization_from(succeeded, response)
```

Comment:
Do we still need succeeded as a parameter here? It doesn't seem like we use it in this method

Reply:
good point, will revise this!